web-common/src/features/dashboards/pivot/FlatTable.svelte

```diff
@@ -0,0 +1,304 @@
+<script lang="ts" context="module">
+  import { writable } from "svelte/store";
+  const columnLengths = writable(new Map<string, number>());
+</script>
+
+<script lang="ts">
+  import ArrowDown from "@rilldata/web-common/components/icons/ArrowDown.svelte";
+  import type { MeasureColumnProps } from "@rilldata/web-common/features/dashboards/pivot/pivot-column-definition";
+  import {
+    calculateColumnWidth,
+    calculateMeasureWidth,
+    COLUMN_WIDTH_CONSTANTS as WIDTHS,
+  } from "@rilldata/web-common/features/dashboards/pivot/pivot-column-width-utils";
+  import Resizer from "@rilldata/web-common/layout/Resizer.svelte";
+  import { modified } from "@rilldata/web-common/lib/actions/modified-click";
+  import type { Cell, Column, HeaderGroup, Row } from "@tanstack/svelte-table";
+  import { flexRender } from "@tanstack/svelte-table";
+  import type { PivotDataRow } from "./types";
+
+  // State props
+  export let assembled: boolean;
```

Comment:
What's this mean?

Reply:
There are multiple stages in a pivot construction. `assembled: true` signifies that all the stages have been completed.